integrations.json

```diff
@@ -5035,6 +5035,33 @@
       }
     ]
   },
+  {
+    "name": "AspNetCore",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "Microsoft.AspNetCore.Http",
+          "type": "Microsoft.AspNetCore.Builder.ApplicationBuilder",
+          "method": "Build",
+          "signature_types": [
+            "System.Void"
+          ],
+          "minimum_major": 2,
+          "minimum_minor": 0,
+          "minimum_patch": 0,
+          "maximum_major": 6,
+          "maximum_minor": 65535,
+          "maximum_patch": 65535
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace.ClrProfiler.Managed, Version=1.28.1.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```

Comment:
Don't forget to update when you rebase 🙂

Reply:
fixed in 46c52128